### ISSUE TITLE
Add addtional react production artifact sourcemaps

### ISSUE THREAD

**Issue by jasikpark** [markerikson/react-prod-sourcemaps#3]
I'm guessing react-dom was the MVP option for testing this out? It would be great to setup sourcemaps for

- react.production.min.js
- react-jsx-runtime.production.min.js

in addition to react-dom

**Comment by markerikson** [comment]
Yeah, easy to throw those in too, and I tried to set up the logic and `assets` folder to support that :)  but `react-dom` was the critical part, since it has all the reconciler logic.

**Comment by JonasBa** [comment]
@markerikson would be nice if we could generate sourcemaps for react-dom/profiling as well. I would be happy to spend some time in order to get that working too

**Comment by markerikson** [comment]
Added `react` and `react-dom.profiling` for 17.0.2, 18.1, and 18.2!

https://github.com/markerikson/react-prod-sourcemaps/releases/tag/v0.2.0

**Comment by markerikson** [comment]
A few weeks ago I did try to build React 16.14.0 as well, but my first attempt generated JS build artifacts that were significantly different.

Looks like this thread explains why:

- https://github.com/facebook/react/issues/27515

16.14.0 is really 16.13.1 , with the JSX runtime artifacts from 17 copied over.
